trade-finance-manager-ui/templates/case/cancellation/reason-for-cancelling.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "index.njk" %}
+
+{% block pageTitle -%}
+  Cancellation - Reason for cancelling
+{%- endblock %}
+
+{% set headingHtml %}
+
+<h1 class="govuk-label-wrapper">
+  <span class="govuk-caption-l">Cancel deal {{ ukefDealId }}</span>
+  <label class="govuk-label govuk-label--l" for="reason-for-cancelling">
+    Reason for cancelling deal (optional)
+  </label>
+</h1>
```

Comment:
This pattern is taken from https://design-system.service.gov.uk/styles/headings/#headings-with-captions